tests/core/test_connector_sql.py

```diff
@@ -14,7 +14,7 @@ class TestConnectorSQL:
 
     @pytest.fixture()
     def connector(self):
-        return SQLConnector()
+        return SQLConnector(config={"sqlalchemy_url": "sqlite:///tmp.db"})
```

Comment:
Can an in-memory db be used here?

```suggestion
        return SQLConnector(config={"sqlalchemy_url": "sqlite://"})
```

Reply:
Oh oops yes! One other small change to make downstream of that, will push it in a sec